addons/website_sale/models/product.py

```diff
@@ -214,7 +214,7 @@ class ProductTemplate(models.Model):
                                         help="The product will be available in each mentioned e-commerce category. Go to"
                                         "Shop > Customize and enable 'E-commerce categories' to view all e-commerce categories.")
 
-    product_template_image_ids = fields.One2many('product.image', 'product_tmpl_id', string="Extra Product Images", copy=True)
+    product_template_image_ids = fields.One2many('product.image', 'product_tmpl_id', string="Extra Product Medias", copy=True)
```

Comment:
Media (plural), also the field product_variant_image_ids on product product can be updated the same.

Reply:
`product_variant_image_ids` still has Images instead of Media in string, but this is not critical if we want to merge this for the bug fix.